pkgs/servers/nfs-ganesha/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig
+, krb5, xfsprogs, jemalloc, dbus, libcap
+, ntirpc, liburcu, bison, flex, nfs-utils
+} :
+
+stdenv.mkDerivation rec {
+  pname = "nfs-ganesha";
+  version = "3.3";
+
+  src = fetchFromGitHub {
+    owner = "nfs-ganesha";
+    repo = "nfs-ganesha";
+    rev = "V${version}";
+    sha256 = "1w48rqrbqah0hnirvjdz8lyr9ah8b73j3cgsppb04gnrmpssgmb6";
+  };
+
+  patches = [ ./sysstatedir.patch ];
```

Comment:
What is the point of this patch? If this is a support for runstatedir, could this patch be opened upstream?

Reply:
Problem is that `@SYSSTATEDIR@` contains the prefix, which is used here for the pid file (that should maybe be runstatedir?) as well as for /var/lib. On anything else than nix/NixOS that might actually be OK. However, for us this means that the daemon tries to create files in the nix store.

Reply:
@jtojnar For now I do not see a possibility to get rid of the patch. Should we move forward with the PR?